ibus.service.in

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=ibus daemon
+Documentation=man:ibus-daemon(1)
+
+[Service]
+Type=dbus
+BusName=org.freedesktop.IBus
+Environment=IBUS_SOCK=%t/ibus.socket
+ExecStart=@bindir@/ibus-daemon --replace --xim --panel disable -a unix:tmpdir=$IBUS_SOCK
```

Comment:
Hello, why should org.freedesktop.IBus.service.in renamed to bus.service.in? the service file in `bus/` is a `dbus-activated`-service. Therefore it should have a FQDN service name. Just `bus.service.in` is not specific enough.I would further go with `ibus.service` as well in the `/etc/systemd/user/`-dir.

For ibus.service you can use `/etc/systemd/user/ibus.service` or `$HOME/.config/systemd/user/ibus.service`.

I will modify the part with `--xim` this makes absolutly sense.